spec/controllers/admin_general_controller_spec.rb

```diff
@@ -5,14 +5,18 @@
 
   describe "when viewing front page of admin interface" do
 
-    render_views
-    before { basic_auth_login @request }
-
     it "should render the front page" do
       get :index
       expect(response).to render_template('index')
     end
 
+    it 'assigns old unclassified requests' do
+      InfoRequest.destroy_all
+      @old_request = FactoryGirl.create(:old_unclassified_request)
+      get :index
+      expect(assigns[:old_unclassified]).to eq([@old_request])
+    end
+
```

Comment:
Looks like this commit should be a fixup to 57c3d829344bb9611427ec5b4a319300c81936d2?